projects/mathjax-lib/README.md

```diff
@@ -1,24 +1,144 @@
-# MathjaxLib
+# mathjax-angular 
+This plugins implements the browser version of [Mathjax v3][1] into [Angular][2]
+## Feature
 
-This library was generated with [Angular CLI](https://github.com/angular/angular-cli) version 12.1.1.
+- Dynamically load MathJax library to your web application.
```

Comment:
rephrase this to our own words

Reply:
any web application
mention that as this lib is developed in angular12, this may not support old ie12 or later version.